doc/audit/ledger.rst

```diff
@@ -0,0 +1,50 @@
+Ledger
+======
+
+The CCF ledger is the persistent replicated append-only record of the transactions that have been executed by the network. It is written by the primary node when a transaction is committed and replicated to all backups which maintain their own duplicated copy. Each node in a network creates and maintains its own local copy of the ledger. Committed entries are always identical, but a node may be more or less up to date, and uncommitted entries may differ.
```

Comment:
This paragraph uses committed in two different ways, first to describe transactions committed to the local KV of the primary, and last to describe transactions that aren't committed according to consensus. We ought to clarify that.

Reply:
Good spot. Changed the first instance of "committed" to "executed".